.doc_gen/metadata/cloudwatch-events_metadata.yaml

```diff
@@ -0,0 +1,82 @@
+                  sdkguide: sdk-for-javascript/v2/developer-guide/cloudwatch-examples-sending-events.html#cloudwatch-examples-sending-events-rules
+                  excerpts:
+                      - description:
+                        snippet_tags:
+                            - cwEvents.JavaScript.cwe.putRule
+    services:
+        - cloudwatch-events
+cloudwatch-events_PutEvents:
+    title: Sending &CWE; events using an &AWS; SDK
+    title_abbrev: Send events
+    synopsis: send &CWElong; events.
+    category:
+    languages:
+        JavaScript:
+            versions:
+                - sdk_version: 3
+                  github: javascriptv3/example_code/cloudwatch-events
+                  sdkguide: sdk-for-javascript/v3/developer-guide/cloudwatch-examples-sending-events.html#cloudwatch-examples-sending-events-putevents
+                  excerpts:
+                      - description: Create the client in a separate module and export it.
+                        snippet_tags:
+                            - cloudwatch.JavaScript.events.createclientv3
+                      - description: Import the SDK and client modules and call the API.
+                        snippet_tags:
+                            - cwEvents.JavaScript.cwe.putEventsV3
+                - sdk_version: 2
+                  github: javascript/example_code/cloudwatch
+                  sdkguide: sdk-for-javascript/v2/developer-guide/cloudwatch-examples-sending-events.html#cloudwatch-examples-sending-events-putevents
+                  excerpts:
+                      - description:
+                        snippet_tags:
+                            - cwEvents.JavaScript.cwe.putEvents
+    services:
+        - cloudwatch-events
+cloudwatch-events_PutTargets:
+    title: Adding a &LAM; function target using an &AWS; SDK
+    title_abbrev: Adding a &LAM; function target
+    synopsis: add an &LAMlong; function target to an &CWElong; event.
+    category:
+    languages:
+        JavaScript:
+            versions:
+                - sdk_version: 3
+                  github: javascriptv3/example_code/cloudwatch-events
+                  sdkguide: sdk-for-javascript/v3/developer-guide/cloudwatch-examples-sending-events.html#cloudwatch-examples-sending-events-targets
+                  excerpts:
+                      - description: Create the client in a separate module and export it.
+                        snippet_tags:
+                            - cloudwatch.JavaScript.events.createclientv3
+                      - description: Import the SDK and client modules and call the API.
+                        snippet_tags:
+                            - cwEvents.JavaScript.cwe.putTargetsV3
+                - sdk_version: 2
+                  github: javascript/example_code/cloudwatch
+                  sdkguide: sdk-for-javascript/v2/developer-guide/cloudwatch-examples-sending-events.html#cloudwatch-examples-sending-events-rules
```

Comment:
This links to a put rule topic, but the example is about setting a function target.

Reply:
fixed